tgui/packages/tgui/interfaces/MatMarket.tsx

```diff
@@ -133,6 +137,13 @@ export const MatMarket = (props) => {
                     >
                       {toTitleCase(material.name)}
                     </Stack.Item>
+                    <Stack.Item
+                      width="10%"
+                      pr="2%"
+                      textColor={material.elastic < 0.33 ? 'red' : material.elastic < 0.66 ? 'orange' : 'green'}
+                    >
+                      Elasticity: <b>{Math.round(material.elastic * 100)}</b>%
```

Comment:
Please not elasticity isn't a percentage that goes from 0->1. It's the actual cost of the item that goes from
 0->that item price
https://github.com/tgstation/tgstation/blob/a70e462e643694d473070a7232004f7811f90e29/code/modules/cargo/exports.dm#L141-L145
If you want a percentage you have to do `(init_cost / cost)` for that export